morf-oracle/src/main/java/org/alfasoftware/morf/jdbc/oracle/OracleDialect.java

```diff
@@ -423,25 +425,21 @@ public Collection<String> dropStatements(View view) {
 
 
   /**
-   * @see org.alfasoftware.morf.jdbc.SqlDialect#viewDeploymentStatementsAsLiteral(org.alfasoftware.morf.metadata.View)
+   * @see org.alfasoftware.morf.jdbc.SqlDialect#getSqlFrom(org.alfasoftware.morf.sql.element.ClobFieldLiteral)
    */
   @Override
-  public AliasedField viewDeploymentStatementsAsLiteral(View view) {
-    final String script = viewDeploymentStatementsAsScript(view);
-
+  protected String getSqlFrom(ClobFieldLiteral field) {
     final int chunkSize = 512;
-    if (script.length() <= chunkSize) {
-      return super.viewDeploymentStatementsAsLiteral(view);
+    if (field.getValue().length() <= chunkSize) {
+      return getSqlFrom((FieldLiteral) field);

```

Comment:
This shoud be `return super.getSqlFrom(field);`
which will effectively call `getSqlFrom((FieldLiteral) field)`.